docs/docs/zh/guide/task/flink.md

```diff
@@ -64,6 +70,14 @@ Flink 任务类型，用于执行 Flink 程序。对于 Flink 节点，worker 
 
 ![demo-flink-simple](/img/tasks/demo/flink_task02.png)
 
+### 执行 FlinkSQL 程序
+
+根据上述参数说明，配置所需的内容即可。
+
+![demo-flink-sql-simple](/img/tasks/demo/flink_sql_test.png)
+
 ## 注意事项：
 
-Java 和 Scala 只是用来标识，没有区别，如果是 Python 开发的 Flink 则没有主函数的 class，其余的都一样。
+- Java 和 Scala 只是用来标识，没有区别，如果是 Python 开发的 Flink 则没有主函数的 class，其余的都一样。
+
+- 使用sql 执行Flink SQL 任务，目前只支持flink1.13及以上版本。
```

Comment:
We should keep the same format with or without spaces. ```使用 SQL 执行 Flink SQL 任务```

Reply:
Thanks a lot, I'll check again